layouts/shortcodes/api.html

```diff
@@ -0,0 +1,23 @@
+
+<tr>
+    <td>
+        <span class="badge bg-secondary">API</span>
+        {{ $engines := .Site.Params.options.engines }}
+        {{ if (.Get "engine") }}
+        {{ $engine := .Get "engine" }}
+        {{ if in $engines $engine }}

```

Comment:
This is repeated in all shorcodes as I couldn't find a way to nest  shortcodes or share functionality.

Reply:
We discuissed some pros and cons.
Dedicated shortcode templates are OK - something specific potentially might be changed in one of them.